proxy/proxy.go

```diff
@@ -515,13 +547,26 @@ func (p *Proxy) do(ctx *context) error {
 		ctx.outgoingDebugRequest = debugReq
 		ctx.setResponse(&http.Response{Header: make(http.Header)}, p.flags.PreserveOriginal())
 	} else {
+		done, allow := p.checkBreaker(ctx)
+		if !allow {
+			return errCircuitBreakerOpen
```

Comment:
Do I read this correct, that we will only log that a circuit breaker is open, but not which one? If a breaker flips open (or more general flips state), I would like to see a log line stating clearly, which breaker for which host goes from which old state into which new state.

Is this not there or am I missing something?